core/new-gui/src/app/dashboard/user/component/user-workflow/ngbd-modal-workflow-executions/workflow-execution-modal.component.ts

```diff
@@ -115,10 +116,17 @@ export class WorkflowExecutionModalComponent implements OnInit, AfterViewInit {
   constructor(
     private workflowExecutionsService: WorkflowExecutionsService,
     private notificationService: NotificationService,
-    private runtimeStatisticsModal: NzModalService
+    private runtimeStatisticsModal: NzModalService,
+    private workflowActionService: WorkflowActionService
   ) {}
 
   ngOnInit(): void {
+    const wid = this.workflowActionService.getWorkflowMetadata()?.wid;
```

Comment:
Please ref to the wid fix in version history: #2499.

Reply:
fixed